contracts/contracts/activities/Activity.sol

```diff
@@ -51,6 +63,7 @@ struct ValidatorClaimProof {
 }
 
 /// The proofs to batch claim validator rewards in a specific subnet
+/// REVIEW(raulk): Delete this type. Make the method just take the subnet ID and the list of claim proofs.
```

Comment:
@raulk I think this type is introduced to handle multiple subnet ids at once. With this structure, might be easier to manage types. Or this requirement is not needed?

Reply:
So there's two levels of batching:
1. Claim rewards corresponding to multiple checkpoint heights for a single subnet at once.
2. Claim rewards from multiple subnets, over multiple checkpoint heights for each subnet, at once.

I would only support (1) and not (2) as:
1. It limits attack surface by constraining interactions to a single subnet within a transaction.
2. It sounds like premature optimization to me, and once introduced it will be impossible to remove even if nobody uses it.

My suggestion is that we descope (2) and limit ourselves to (1). With that in mind, we can kill the extra type and simply have three arguments:

- subnet ID
- array of checkpoint heights
- array of claims